src/lib/timers/ingress_drop_monitor.lua

```diff
@@ -65,6 +79,8 @@ function IngressDropMonitor:jit_flush_if_needed ()
    if now() - self.last_flush < self.wait then return end
    self.last_flush = now()
    self.last_value[0] = self.current_value[0]
+   counter.set(self.shm['ingress_packet_drop'], self.last_value[0])
```

Comment:
nit: use direct access where possible (in this case `self.shm.ingress_packet_drop`)